conn.go

```diff
@@ -667,6 +667,8 @@ func (c *Conn) handleIncomingPacket(buf []byte) (*alert, error) {
 	case *alert:
 		c.log.Tracef("<- %s", content.String())
 		if content.alertDescription == alertCloseNotify {
+			// Respond with a close_notify [RFC5246 Section 7.2.1]
+			c.notify(alertLevelWarning, alertCloseNotify) //nolint
```

Comment:
are you adding `//nolint` to skip the error processing? You can do `_ = c.notify` instead

Reply:
agree with this.
I'll also fix one in `close()` where I copied it from.